apps/omg_eth/lib/omg_eth/root_chain.ex

```diff
@@ -376,7 +375,7 @@ defmodule OMG.Eth.RootChain do
     )
   end
 
-  defp decode_exit_started(log) do
+  def decode_exit_started(log) do
```

Comment:
if you want to test this function you should extract it to a separate module so that it becomes available for unit testing... it seems like it's not used >in code<

Reply:
Updated in 84d6fc67de63477e104fcce55a7d83c316523957